examples/overview/build/index.main.mjs

```diff
@@ -67,17 +67,13 @@ export async function Alice(ctcTop, interact) {
       
       const {data: [v69], secs: v71, time: v70, didSend: v28, from: v68 } = txn1;
       
-      sim_r.txns.push({
```

Comment:
It is fine to commit this this time, but FYI most of the time the changes in this are just to the bytecode and don't need to be updated